enterprise/server/remote_execution/execution_server/execution_server.go

```diff
@@ -1007,31 +1007,44 @@ func (s *ExecutionServer) PublishOperation(stream repb.Execution_PublishOperatio
 			return err
 		}
 
+		if op.GetName() == "" {
+			return status.InvalidArgumentError("Operation.name must be non-empty")
+		}
```

Comment:
I trust you've checked and this is safe?

Reply:
by the way, if taskID is empty, I would expect https://github.com/buildbuddy-io/buildbuddy/blob/d06497cd4e1aa94233a327df1d70c844192ae0ab/enterprise/server/remote_execution/execution_server/execution_server.go#L1096 to return an error, and it never does: https://console.cloud.google.com/logs/query;query=SEARCH%2528%22Unparsable%20resource%20name%22%2529;cursorTimestamp=2024-12-03T15:26:10.559419765Z;duration=P30D?project=flame-build&inv=1&invt=AbjUVw